packages/kbn-utils/README.md

```diff
@@ -0,0 +1,3 @@
+# @kbn/utils
```

Comment:
who is going to be an owner of this package?

Reply:
https://github.com/elastic/kibana/pull/76518/commits/c9eef0eb0293042b98a562a37cb8d47a519a4b1c#diff-5cfb934ca0ec4c1743905a51dc428b84R131